prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml

```diff
@@ -12,7 +12,7 @@ presubmits:
       - image: gcr.io/compute-image-tools/daisy:latest
```

Comment:
I guess we also want to change the image, right?

Reply:
done